### DISCUSSION TITLE
couldnt reach local plex server at server address: http://localhost:32400/web - or this library really is empty

### DISCUSSION THREAD

**Discussion by deltaxsingh** [itsToggle/plex_debrid#94] [General]
Really no idea why this would be the case. 

I can access the server just fine from my PC yet I keep getting this error. 

All media shows up in the folders, the driver letter is also correct, inside Plex I can also see the media folders. 

I'm on Windows. There is content already inside the specified folders as well, most definitely not empty. 

Any light to fix this issue?

**Comment by itsToggle** [comment]
The server address needs to be in the format: `http://localhost:32400` without the `/web` at the end :)

**Reply by deltaxsingh** [reply]
Damn, tried that too. No dice ):

EDIT: It worked! For some reason, changing the settings inside the python script didn't stick. I went into the .json and manually changed the server address. Thanks @itsToggle

**Reply by beljim** [reply]
Try http://127.0.0.1:32400

**Reply by itsToggle** [reply]
Nice :+1:  Settings are only saved once you back out to the main menu again.

**Comment by ghost** [comment]
> Really no idea why this would be the case.
> 
> I can access the server just fine from my PC yet I keep getting this error.
> 
> All media shows up in the folders, the driver letter is also correct, inside Plex I can also see the media folders.
> 
> I'm on Windows. There is content already inside the specified folders as well, most definitely not empty.
> 
> Any light to fix this issue?

You will probably get a lot of these issues for the next day or 2 due to the traffic plex are receiving to reset passwords due to the data breach, i was getting the same error a lot too i just kept retrying at different times of the day and it worked in the end, hope this helps
